### DISCUSSION TITLE
PS 10: Problem 2.2 b)

### DISCUSSION THREAD

**Discussion by AlexanderZhurauski** [spamegg1/Math-for-CS-solutions#3] [Ideas]
The question is as follows: "How many ways are there to order the 26 letters of the alphabet so that there are at least two consonants
immediately following each vowel?"
Doesn't the solution presented just count the number of ways to order 26 letters so that there are EXACTLY two consonants immediately following each vowel? To my mind it should be more like:
(all ways to order 26 letters) - (ways to order 26 letters so that rhere are 0 or 1 consonants immediately follow each vowel)
Sorry if I'm having a brain fart and am completely wrong

**Comment by spamegg1** [comment]
That way of doing is also fine.

There can be more than two consonants because of the ending of the problem:

> Now there are 11 consonants left, along with the 5 “one vowel + two consonant” groups, a total of 16 groups that we can order however we want.

So, some of the remaining 11 consonants can go after the "one vowel + two consonant" groups, or not.

Like so:
```
(atk) c b (ezx) (iqy) l s m ...
```

If you are struggling, you should try it out on a much smaller set of letters, and hand-count everything by writing them down literally one by one. Then slightly increase the size, see if you can generalize.

**Reply by AlexanderZhurauski** [reply]
Ah, I understand now! For some reason it entirely escaped me that a subset of these 11 remaining consonants could end up after a "VCC" group therefore increasing the number of consonants following that particular vowel. Thank you for clarifying!
